plugin/builtin/sortordertransformer/SortOrderTransformer.go

```diff
@@ -0,0 +1,244 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:generate pluginator
+package main
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/kustomize/api/resmap"
+	"sigs.k8s.io/kustomize/api/resource"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kyaml/resid"
+	"sigs.k8s.io/yaml"
+)
+
+// Sort the resources using a customizable ordering based of Kind.
+// Defaults to the ordering of the GVK struct, which puts cluster-wide basic
+// resources with no dependencies (like Namespace, StorageClass, etc.) first,
+// and resources with a high number of dependencies
+// (like ValidatingWebhookConfiguration) last.
+type plugin struct {
+	SortOptions *types.SortOptions `json:"sortOptions,omitempty" yaml:"sortOptions,omitempty"`
+}
+
+// noinspection GoUnusedGlobalVariable
```

Comment:
The PR you rebased on removed this line from the other transformer (it isn't handled by pluginator anymore and gets copied, without the line below, to the generated file). Not a big deal, but since you'll need to push again anyway, can you please remove it and rerun `make generate-kustomize-builtin-plugins` to remove it from the generated file too?